scripts/qa_crowbarsetup.sh

```diff
@@ -3256,6 +3261,37 @@ function onadmin_qa_test()
     return $ret
 }
 
+function onadmin_run_cct()
+{
+
+    addcloudrubygemrepo
+    # - install cct dependencies
+    ensure_packages_installed git-core gcc make ruby2.1-devel
+
+    # - default checkout location for mkcloud based setups:
+    mkdir -p /root/github.com/SUSE-Cloud
+    pushd /root/github.com/SUSE-Cloud/
+    git clone https://github.com/SUSE-Cloud/cct.git
+    cd cct
+    bundle install
+
+    if [[ -n $cct_tests ]]; then
+        local IFS
+        IFS='+'
+        for test in $cct_tests; do
+            bundle exec rake $test
+            if [[ $? -gt 0 ]]; then
+                ret=$?
```

Comment:
:-1: in this place, $? is already the return code from the test above. just use if bundle exec ... ; then